docs/yuidoc-p5-theme-src/scripts/tpl/menu.html

```diff
@@ -1,3 +1,10 @@
+<p>
+  <small>
+    Can't find what you're looking for? You may want to check out
+    <a href="#/libraries/p5.dom">p5.dom</a> or
+    <a href="#/libraries/p5.sound">p5.sound</a>.
+  </small>
+</p>
```

Comment:
I *think* that this `menu.html` template is only used from the main page of the reference documentation... let me know if I'm wrong though!